src/modules/data/Height.ts

```diff
@@ -30,12 +30,11 @@ export class Height
      * Construct
      * @param value - The block height
      */
-    constructor (value?: UInt64)
+    constructor (value: bigint);
+    constructor (value: UInt64);
+    constructor (value: any)
```

Comment:
How about this?
```suggestion
    constructor (value: bigint | string | UInt64)
    {
        if (typeof value == "bigint")
        {
            this.value = UInt64.fromString(value.toString());
        }
        else if (typeof value === "string")
        {
            this.value = UInt64.fromString(value);
        }
        else
        {
            this.value = new UInt64(value);
        }
    }
```

Reply:
Good suggestion, I committed a slightly modified version.